escapades_frontend/src/components/HomeImg/CoverCard.module.scss

```diff
@@ -0,0 +1,8 @@
+.imgContainer {
+  @apply relative w-screen h-[33vh] sm:h-[66vh] bg-cover;
+
+  &__title {
+    @apply absolute top-1/4 sm:top-1/3 left-1/2 z-[9998] text-white text-base sm:text-3xl xl:text-5xl;
+    transform: translate(-50%, 0);
+  }
+}
```

Comment:
Nice here ✨

However, for the naming convention, it is a good practice to name the folder with the same name as the component.
Here I think we may call it HomeBanner, what do you think?

Reply:
lets go for HomeBanner